build/make/k8s.mk

```diff
@@ -51,41 +65,44 @@ k8s-delete: k8s-generate $(K8S_POST_GENERATE_TARGETS) ## Deletes all dogu relate
 K8S_POST_GENERATE_TARGETS ?=
 # The additional targets executed before the generate target, executed before each apply and delete.
 K8S_PRE_GENERATE_TARGETS ?= k8s-create-temporary-resource
+
 .PHONY: k8s-generate
 k8s-generate: $(K8S_RESOURCE_TEMP_FOLDER) $(K8S_PRE_GENERATE_TARGETS) ## Generates the final resource yaml.
 	@echo "Applying general transformations..."
 	@sed -i "s/'{{ .Namespace }}'/$(K8S_CURRENT_NAMESPACE)/" $(K8S_RESOURCE_TEMP_YAML)
-	@yq -i e "(select(.kind == \"Deployment\").spec.template.spec.containers[]|select(.image == \"*$(ARTIFACT_ID)*\").image)=\"$(IMAGE)\"" $(K8S_RESOURCE_TEMP_YAML)
+	@yq -i e "(select(.kind == \"Deployment\").spec.template.spec.containers[]|select(.image == \"*$(ARTIFACT_ID)*\").image)=\"$(IMAGE_DEV)\"" $(K8S_RESOURCE_TEMP_YAML)
 	@echo "Done."
 
 .PHONY: k8s-apply
-k8s-apply: k8s-generate $(K8S_POST_GENERATE_TARGETS) ## Applies all dogu related resources from the K8s cluster.
-	@echo "Apply new dogu resources..."
+k8s-apply: k8s-generate $(K8S_POST_GENERATE_TARGETS) ## Applies all generated K8s resources to the current cluster and namespace.
+	@echo "Apply generated K8s resources..."
 	@kubectl apply -f $(K8S_RESOURCE_TEMP_YAML)
 
 ##@ K8s - Docker
 
 .PHONY: docker-build
-docker-build: check-k8s-image-env-var ## Builds the docker image of the k8s app.
+docker-build: check-k8s-image-env-var ## Builds the production docker image of the K8s app.
```

Comment:
To what does `production` refer here? Is there a `development` docker image?

Reply:
done.